documentation/configuration/basic.md

```diff
@@ -36,7 +36,7 @@ None of the items are optional, you need to accept everything before you can set
 
 ## Device registration
 
-HACS uses a device OAuth flow for authentication against the GitHub API.
+HACS uses a device OAuth flow for authentication against the GitHub API (also see [Why do i need a GitHub Account?](/docs/faq/github_account)).
```

Comment:
Most folks does not care.
Just make `HACS uses a device OAuth flow for authentication` clickable instead

Reply:
The faq entry has nothing to do with the OAuth flow specific, but why access is needed in the first place. Better to seperate and make it explicit.